datahub-web-react/src/app/entity/shared/entity/EntityActions.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { Button, message } from 'antd';
+import { LinkOutlined } from '@ant-design/icons';
+import { SearchSelectModal } from '../components/styled/search/SearchSelectModal';
+import { useEntityRegistry } from '../../../useEntityRegistry';
+import { EntityCapabilityType } from '../../Entity';
+
+export enum EntityActionItem {
+    /**
+     * Batch add a Glossary Term to a set of assets
+     */
+    BATCH_ADD_GLOSSARY_TERM,
+    /**
+     * Batch add a Domain to a set of assets
+     */
+    BATCH_ADD_DOMAIN,
+}
+
+interface Props {
+    actionItems: Set<EntityActionItem>;
+    refetchForEntity?: () => void;
+}
+
+function EntityActions(props: Props) {
+    // eslint ignore react/no-unused-prop-types
+    const entityRegistry = useEntityRegistry();
+    const { actionItems, refetchForEntity } = props;
+    const [isBatchAddGlossaryTermModalVisible, setIsBatchAddGlossaryTermModalVisible] = useState(false);
+    const [isBatchSetDomainModalVisible, setIsBatchSetDomainModalVisible] = useState(false);
+
+    // eslint-disable-next-line
+    const batchAddGlossaryTerms = (entityUrns: Array<string>) => {
```

Comment:
usually in ts you just say `string[]` for arrays. that's probably why you're seeing a lint warning. https://stackoverflow.com/questions/36842158/arraytype-vs-type-in-typescript

Reply:
ah or is it bc `entityUrns` isn't being used? either way `string[]` would be more idiomatic

Reply:
Yeah entityUrns not being used :( 

It's because this functionality is not yet accessible / fully implemented. Working on the backend APIs then coming back!